config/events/config.go

```diff
@@ -0,0 +1,43 @@
+package events
+
+import (
+	"github.com/upbound/upjet/pkg/config"
+)
+
+// Configure adds configurations for events group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_cloudwatch_event_permission", func(r *config.Resource) {
+		r.References["event_bus_name"] = config.Reference{
+			Type:              "github.com/upbound/provider-aws/apis/events/v1beta1.Bus",
+			RefFieldName:      "EventBusNameRefs",
+			SelectorFieldName: "EventBusNameSelector",
+		}
+		r.UseAsync = true
+	})
+	p.AddResourceConfigurator("aws_cloudwatch_event_rule", func(r *config.Resource) {
+		r.References["event_bus_name"] = config.Reference{
+			Type:              "github.com/upbound/provider-aws/apis/events/v1beta1.Bus",
+			RefFieldName:      "EventBusNameRefs",
+			SelectorFieldName: "EventBusNameSelector",
+		}
+		r.UseAsync = true
```

Comment:
Please see above for a discussion:
```suggestion
			Type:              "Bus",
		}
```